docs/UserGuide.md

```diff
@@ -295,6 +295,40 @@ Examples:
 * `elist` followed by `edelete 2` deletes the 2nd event in the event list.
 * `efind Beach cleaning` followed by `edelete 1` deletes the 1st event in the results of the `find` command (tentative feature)
 
+### Edit the details of an event: `eedit`
+
+Volunteer coordinators can edit the details of the events.
+
+Format: `eedit EVENT_Id [n/EVENT_NAME] [r/ROLES_NEED]... [sd/START_DATETIME] [ed/END_DATETIME] [l/LOCATION] [dsc/DESCRIPTION] [m/MATERIALS_AND_LOGISTICS_NEEDED]... [b/BUDGET]`
```

Comment:
now the `EVENT_Id` has been updated to `EVENT_INDEX` for clarity purposes

Reply:
also very minor but `[r/ROLES_NEED]` should be` [r/ROLES_NEEDED]` HAHAHAH

Reply:
also u might have missed out on max volunteer count parameter

Reply:
Ok fixed.